examples/with-mocha/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "with-mocha",
+  "version": "0.0.0",
+  "scripts": {
+    "test": "cross-env NODE_ENV=test mocha --timeout 10000 src/**/*.test.js",
```

Comment:
Could it be that the test files aren't included in the commit?

Reply:
I sweat I've used git before... promise.